src/app/clusters/door-lock-server/door-lock-server.h

```diff
@@ -180,6 +180,17 @@ class DoorLockServer
         return GetFeatures(endpointId).Has(DoorLockFeature::kUser) && SupportsAnyCredential(endpointId);
     }
 
+    inline void ResetWrongCodeEntryAttempts(chip::EndpointId endpointId)
```

Comment:
Needs documentation.  And why is the implementation inline?

Reply:
done